packages/dynamite/dynamite_petstore_example/pubspec.yaml

```diff
@@ -13,6 +13,8 @@ dependencies:
     git:
       url: https://github.com/nextcloud/neon
       path: packages/dynamite/dynamite_runtime
+  meta: ^1.11.0
+  universal_io: ^2.2.2
```

Comment:
We won't publish this so we don't exactly need to care about the constraints, but someone might copy them so it's best to set the correct ones.